apps/web/app/routes/stacking/stacking.tsx

```diff
@@ -17,13 +22,14 @@ export function meta({}: Route.MetaArgs) {
 
 export default function EarnStackingRoute({ params }: Route.ComponentProps) {
   const poolSlug = params.slug as PoolSlug;
```

Comment:
Can we add zod validators, and ensure we don't get invalid values being passed in? A user can go to any route and the page will break

Reply:
added zod validation, though isn't it expected behaviour that page breaks? we just need to add err component where pass err msg

Reply:
expected behaviour is that we go to 404 page, not breaks